lua/mason-registry/buf-language-server/init.lua

```diff
@@ -0,0 +1,14 @@
+local Pkg = require "mason-core.package"
+local go = require "mason-core.managers.go"
+
+return Pkg.new {
+    name = "buf-language-server",
+    desc = [[`bufls` is a prototype for the beginnings of a Protobuf language server compatible with Buf modules and workspaces.]],
+    homepage = "https://github.com/bufbuild/buf-language-server",
+    languages = { Pkg.Lang.Protobuf },
+    categories = { Pkg.Cat.LSP },
+    install = go.packages {
+        "github.com/bufbuild/buf-language-server/cmd/bufls@latest",
```

Comment:
Version is added dynamically during installation, to allow custom versions

```suggestion
        "github.com/bufbuild/buf-language-server/cmd/bufls",
```